Configuration/FlexForms/AccountFavorite.xml

```diff
@@ -14,7 +14,7 @@
 						<allowed>pages</allowed>
 						<size>1</size>
 						<maxitems>1</maxitems>
-						<minitems>1</minitems>
+						<minitems>0</minitems>
```

Comment:
Before this settings had to be set. Now it's optional because you can predefine this at ts / constant editor